listener/wal_transaction.go

```diff
@@ -217,7 +218,12 @@ func (w *WalTransaction) CreateActionData(
 			valueType: rel.Columns[num].valueType,
 			isKey:     rel.Columns[num].isKey,
 		}
-		column.AssertValue(row.Value)
+		if row.IsUnchangedToastedValue && len(oldRows) > num {
+			oldColumns[num].isUnchangedToastedValue = true
```

Comment:
nit: what do you think of lifting this outside of the branch and doing `oldColumns[num].isUnchangedToastedValue = row.IsUnchangedToastedValue`, so regardless of identity we have the full description of unchange toasted.

Reply:
We can only access `oldColumns[num]` if `len(oldRows) > num`, so I can't lift this outside the branch.

I see what you're saying though and I figured out a different way to always set `unchangedToastedValues` here: [`10a4a22`](https://github.com/gadget-inc/wal-listener/pull/22/commits/10a4a221a6378f3117eadcb03471d5ae21e73354)

Reply:
Right, should have been clearer that the main thing was that, even without `dataOld`, can we relay the unchanged toasted-ness in the new data thing.

You figured out all out 🙌